torch/testing/_internal/distributed/multi_threaded_pg.py

```diff
@@ -341,6 +341,12 @@ def _reduce_scatter_base(self, output_tensor, input_tensor, opts=AllgatherOption
         tensor_list = list(torch.chunk(input_tensor, self._world_size))
         return self.reduce_scatter([output_tensor], [tensor_list], opts)
 
+    def allgather_into_tensor_coalesced(self, output_tensor_list, input_tensor_list):
```

Comment:
should it be a private API in MTPG since its a private API for NCCL?

Reply:
This is the name of the method in the PG api, we could fix it to be private.